usr/src/pkg/transforms/publish

```diff
@@ -32,19 +32,19 @@
 # This is somewhat nonintuitive.  In English:
 #
 #	For any action of type "set",
-#	if the "name" property matches the regex "variant.arch",
-#	and *every* occurrence of the "value" property does not match
+#	if the "name" property matches the regex "org.illumos.build-on",
+#	and *every* occurrence of the "value" property does *not* match
 #		$ARCH, then don't publish it.
 #
 # ...in other words, if *any* occurrence of "value" matches $(ARCH),
 # then the negative lookahead assertion in the regex will fail, and we
 # won't abort based on the action.
 #
-<transform set name=variant.arch value=^(?!$(ARCH)).+$ -> abort>
+<transform set name=org.illumos.build-on value=^(?!$(ARCH)).+$ -> abort>
```

Comment:
We can probably drop the leading `^` as the regexp is anchored at the beginning by default.

Reply:
I think we can, but I'd prefer not to, since apparently it required that big comment in the first place, anything that pins it down explicitly seems best?